src/components/Button.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface Props {
```

Comment:
Would be nice to have a click handler here

Reply:
I have added a simple click handler that currently runs a `handleClick` event in the parent `SummaryCard` component and simply console logs a message

Reply:
In terms of naming, I would start the events with the prefix on. handleClick would be how I would name a function that is assigned to onClick.